### PR TITLE
De best etay

### PR DESCRIPTION
**Pull request by etayruben**

### CODE REVIEW

File: yaml/Etay/yaml_exercise_Etay.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# string
+name: Etay
+# number 
+age: 28
```
Comment (omershilohpsi) [inline yaml/Etay/yaml_exercise_Etay.yaml:4]: You are not 28

File: yaml/Etay/yaml_exercise_Etay.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+isPerson: true
+# array 
+food: ['GF pizza','GF beer', 'GF schnitzel']
+# array option 2
+friends:
```
Comment (omershilohpsi) [inline yaml/Etay/yaml_exercise_Etay.yaml:9]: Are these your real friends?

File: yaml/Etay/yaml_exercise_Etay.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+friends: 
+   - name: Eli
+     age: 28
+     isPerson: true
+   - name: E-T
+     age: 101
+     isPerson: false
+# object / map
+family:
+     Alon:
```
Comment (omershilohpsi) [inline yaml/Etay/yaml_exercise_Etay.yaml:18]: Missing dateofbirth in family

File: yaml/Etay/yaml_exercise_Etay.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+       isOlder: true
+     papa:
+       kids: 3
+       age: 60
+       isOlder: true
+favorite_movies:
+  - shrek
+  - shrek 2
+  - shrek 3
+summary:
```
Comment (omershilohpsi) [inline yaml/Etay/yaml_exercise_Etay.yaml]: Missing multiline

**Review by omershilohpsi — CHANGES_REQUESTED** [review]

**Review by ley3103 — COMMENTED** [review]
its not good

File: yaml/Etay/yaml_exercise_Etay.yaml (hunk 4 above)
Comment (noamu111) [reply yaml/Etay/yaml_exercise_Etay.yaml]: yeah etay you are bad

**Review by omershilohpsi — APPROVED** [review]

**Pull request merged**